features/GEM111_No_duplicated_points_within_a_polyloop_or_polyline.feature

```diff
@@ -0,0 +1,23 @@
+@implementer-agreement
+@GEM
+Feature: GEM111 - No duplicated points within a polyloop or polyline
+
+  Scenario: Agreement on no duplicated points within a polyloop
+
+      Given A file with Schema Identifier "IFC2X3"
+      And An IfcPolyLoop
+      Then The IfcPolyLoop must have no duplicate points
+
+  Scenario: Agreement on no duplicated points within a polyline
+
+      Given A file with Schema Identifier "IFC2X3"
+      And An IfcPolyLine
+      And IfcPolyLine first and last point are different
```

Comment:
Also we should remove IfcPolyline from this step because later we'll probably make it so that it should also be applicable to IfcIndexedPolyCurve (an entity added in IFC4 which is similar in usage, just didn't exist yet at the time this impl agreement was written)

Reply:
If I remove `And An IfcPolyLine` I get some errors, as some unexpected instances are in the context. 

Would you be ok, if I simply added another scenario, like the one below? 

```
  Scenario: Agreement on no duplicated points within a polyline

      Given An IfcIndexedPolyCurve 
      Given IfcIndexedPolyCurve forms an open curve
      Then The IfcIndexedPolyCurve must have no duplicate points
```

If not, I will find a way around this.

Reply:
Also, perhaps if we're including IfcIndexedPolyCurve (a polycurve?), would it be appropriate to change the lingo around this rule, e.g.  from `Feature: GEM111 - No duplicated points within a polyloop or polyline` to `Feature: GEM111 - No duplicated points within a polyloop, polyline or polycurve` ?

Reply:
Used a pronoun instead of a specified object, e.g.:
```

      Given An IfcPolyLine
      And It forms an open curve
      Then It must have no duplicate points
```